src/Qir/Controller/Executable/ClangClient.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine.Invocation;
+using System.Threading.Tasks;
+using Microsoft.Quantum.Qir.Utility;
+
+namespace Microsoft.Quantum.Qir.Executable
+{
+    public class ClangClient : IClangClient
+    {
+        private const string LinkFlag = " -l ";
```

Comment:
I believe the `-l` flag is meant to be used with no space between "l" and the name of the library.

Reply:
It works either way. It seemed to cause errors on Windows when I did it as you suggest, but adding the space (which is actually documented as well) worked.